drivers/mtd/ubi/debug.c

```diff
@@ -556,7 +556,7 @@ int ubi_debugfs_init_dev(struct ubi_device *ubi)
 						   mode, d->dfs_dir,
 						   (void *)ubi_num, &dfs_fops);
 
-	debugfs_create_file("detailed_erase_block_info", S_IRUSR, d->dfs_dir,
+	debugfs_create_file("detailed_erase_block_info", S_IRUGO, d->dfs_dir,
```

Comment:
Why is this needed on bluefin?

Reply:
@chaitu236 I tried looking around and discovered an archived escalation ([link](https://emerson.sharepoint.com/sites/TM-industrial-embedded-pse/SitePages/Track%20sbRIO-9651%20SSD%20Health.aspx)) about tracking flash memory health. My understanding is that the original intent of the commit is to allow customers to access `detailed_erase_block_info` through all interfaces available to them (which they otherwise would not be able to due to file permission issues).

Given that code search did not locate any internal components which pull data from this file, I'm now inclined to believe that this commit is not specific to Bluefin. Would you advise I remove this commit?

Reply:
Not sure if there'd be security concerns with making this file more permissive, but if upstream has chosen S_IRUSR, I'd prefer to keep it that way, especially since we haven't had this commit on x64 targets for many years now and no one complained.

But I'd leave it to @gratian to decide it.

Reply:
> Given that code search did not locate any internal components which pull data from this file, I'm inclined to believe that this commit is not specific to Bluefin, nor do any Bluefin components depend on it. Would you advise I remove this commit?

Yes. Please drop this commit.

Nothing uses it - there's no reason to carry an out-of-tree commit with potential security implications. An end-user can use appropriate privileged user accounts to access the erase block information and/or modify the access permissions themselves (in user-space) if so desired.

Reply:
Got it. Dropped.